papaparse.js

```diff
@@ -1086,7 +1095,7 @@
 			if (_config.skipEmptyLines)
 			{
 				for (var i = 0; i < _results.data.length; i++)
-					if (_results.data[i].length === 1 && _results.data[i][0] === '')
+					if (testEmptyLine(_results.data[i], _config.delimiter))
```

Comment:
So, I feel like, if we do it this way, since it's post parsing you only need to test that all values are empty in the data array. If you use the regex, then it should be on the raw content if it's available.

Reply:
So do we want to let through values that have whitespace, delimiters that were escaped in a quoted field, escaped quotes, and that kind of flotsam? So like if we do something (in ES5 of course) like this:

```
if (_config.skipEmptyLines) {
  _results.data = _results.data.filter(row => row.some(cell => cell !== ''))
}
```

It would behave very differently from what the current feature does.

Reply:
Interesting, we should stripe whitespace in this test, but I think escaped values should not be part of this. `row.join('').trim() === ''` might be the thing to test in greedy mode.